gnome-image-installer/pages/finished/gis-finished-page.c

```diff
@@ -212,10 +215,73 @@ gis_finished_page_shown (GisPage *page)
     }
 }
 
+static void
+write_unattended_config (GisFinishedPage *self)
+{
+  GFile *image_file = NULL;
+  GFile *image_dir = NULL;
+  g_autoptr(GFile) unattended_ini_file = NULL;
+  g_autofree gchar *unattended_ini = NULL;
+  const gchar *locale = setlocale (LC_ALL, NULL);
+  g_autofree gchar *image = NULL;
+  UDisksBlock *block = NULL;
+  g_autofree gchar *vendor = NULL;
+  g_autofree gchar *product = NULL;
+  g_autofree gchar *backup = NULL;
+  g_autoptr(GError) error = NULL;
+  GtkMessageType message_type = GTK_MESSAGE_INFO;
+  const gchar *message = NULL;
+  g_autofree gchar *secondary = NULL;
+  GtkWidget *dialog = NULL;
+
+  /* No-op if an error has occured. */
+  if (gis_store_get_error ())
```

Comment:
Should we give the user some feedback in this case, like playing a ‘bzzt’ sound (in response to Ctrl+U being pressed)?

Reply:
You're joking about the sound but I would like to draw your attention to https://github.com/endlessm/eos-shell/issues/4765

Reply:
I’m actually not joking about the sound, although I probably should have better described it as ‘error bell’ or ‘terminal bell’.